sites/funmooc/src/backend/base/context_processors.py

```diff
@@ -0,0 +1,63 @@
+"""
+Template context_processors
+"""
+import json
+
+from django.conf import settings
+from django.utils.translation import get_language_from_request
+
+from richie.apps.courses.models import Organization
+
+
+def marketing_metas(request):
+    """
+    Context processor to add all information required by Marketing frontend scripts.
+    """
+
+    page = request.current_page or None
+    language = get_language_from_request(request, check_path=True)
+
+    def get_organizations():
+        """
+        Return all organizations linked to the current page via an organization plugin
+        in any of the placeholders on the page.
+        """
+        organizations = []
+
+        # We need to be aware if an organisation is linked to the current page.
+        # This is the case for the organization_detail page where
+        # there is no organization plugin but the page has an organization attribute.
+        if hasattr(page, "organization"):
+            organizations.append(page.organization.code)
```

Comment:
In this case, I would just return `page.organization.code`. Why should we continue looking?

Reply:
In case of organization group, may be a page organization could have organization plugins and a organization object attached?